src/controllers/QueryRunner.ts

```diff
@@ -58,6 +60,8 @@ export default class QueryRunner {
         this._uri = _ownerUri;
         this._title = _editorTitle;
         this._isExecuting = false;
+        this.batchSets = [];
```

Comment:
>batchSets [](start = 13, length = 9)

should this _batchSets to be consistent with all the other fields be initialized, or is there a reason to use the setter?